compiler/transforms/schedule_memref_linalg.py

```diff
@@ -141,6 +142,25 @@ def match_and_rewrite(self, op: memref_stream.GenericOp, rewriter: PatternRewrit
                 AffineMap(6, 0, (M * 8 + m, N * 8 + n)),
             ]
             template_bounds = (None, None, None, 8, 8, 8)
+        elif op.library_call.data == "snax_gemmx":
+            # TODO: move templates to accelerator definitions
+            if isinstance(op.body.block.first_op, QMacOp):
+                # gemm
+                M, N, K, m, n, k = (AffineDimExpr(i) for i in range(6))
+                template = [
+                    AffineMap(6, 0, (M * 8 + m, K * 8 + k)),
+                    AffineMap(6, 0, (K * 8 + k, N * 8 + n)),
+                    AffineMap(6, 0, (M * 8 + m, N * 8 + n)),
+                ]
+                template_bounds = (None, None, None, 8, 8, 8)
+            else:
+                # rescale function of gemmx
+                M, K, m, k = (AffineDimExpr(i) for i in range(4))
+                template = [
+                    AffineMap(4, 0, (M * 8 + m, K * 8 + k)),
+                    AffineMap(4, 0, (M * 8 + m, K * 8 + k)),
+                ]
+                template_bounds = (None, None, 8, 8)
```

Comment:
Could you already write them to be movable?

Reply:
What does this mean?

Reply:
oeps